developer-docs-site/docs/guides/get-test-funds.md

```diff
@@ -44,7 +44,9 @@ Note that you may create specific account types by passing the `--profile` argum
   ```shell
   Enter your private key as a hex literal (0x...) [Current: None | No input: Generate new key (or keep one if present)]
   ```
-  Hit enter to generate a new key.
+  Either:
+  * Hit enter to generate a new key.
+  * Enter an existing private key, for example [found in your wallet](https://petra.app/docs/use).
```

Comment:
This link isn't particularly great. It drops users off in a massive page far from the actual content. I also don't think that users will need this level of guidance. If they do, we probably shouldn't be advising them without more around private key management.